src/Modules/Scheduler.asm

```diff
@@ -21,10 +21,23 @@
 ; You should have received a copy of the GNU General Public License
 ; along with Bluejay.  If not, see <http://www.gnu.org/licenses/>.
 ;
+;**** **** **** **** **** **** **** **** **** **** **** **** ****
+
+; Bitmask to get the remainder of dividing a number by 8
+MASK_MOD_8          EQU 007h
+
+; Mask to get bits 5, 4, and 3 of a number
+MASK_BITS_543       EQU (007h SHL 3)
+
+; Scheduler counter values for steps 1, 3, 5, and 7
+ODD_STEP_1          EQU (001h SHL 3)
```

Comment:
Maybe just rename to `STEP_1` (3, 5)?

Also `MASK_BITS_543` - can we find a more descriptive name?

Reply:
I added more descriptive names